packages/core-utils/src/app/test-utils.ts

```diff
@@ -56,18 +56,19 @@ export class TestUtils {
   }
 
   public static async assertRevertsAsync(
-    revertMessage: string,
-    func: () => Promise<any>
+    func: () => Promise<any>,
+    revertMessage: string
   ): Promise<void> {
     let succeeded = true
     try {
       await func()
       succeeded = false
     } catch (e) {
       if (e instanceof Error) {
-        assert.equal(
-          e.message,
-          `VM Exception while processing transaction: revert ${revertMessage}`
+        const expectedMessage: string = `VM Exception while processing transaction: revert ${revertMessage}`
+        assert(
+          e.message.startsWith(expectedMessage),
+          `Function did not throw expected error. Expected: ${expectedMessage}*, Received: ${e.message}`
```

Comment:
Just making the revert message check be a "starts with" check so we can optionally not match revert message or not match complete revert message.